### PR TITLE
Skip MVN operation on constant branches

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Fixed the error message: `Failed to find reference implementation for Div_1354 Layer with MVN Type on constant propagation`

### Tickets:
 - https://github.com/openvinotoolkit/openvino/issues/5320

### CODE REVIEW

File: inference-engine/src/legacy_api/src/graph_transformer.cpp [hunk 1]
```diff
@@ -221,6 +221,7 @@ static std::vector<std::string> skipConstInfer = {
     "Squeeze",
     "TensorIterator",
     "LSTMSequence",
+    "MVN"
```
Comment (ilya-lavrenov) [inline inference-engine/src/legacy_api/src/graph_transformer.cpp:224]: Coming back to original root cause - why ngraph does not fold this constant?

Comment (ilyachur) [reply inference-engine/src/legacy_api/src/graph_transformer.cpp:224]: I faced with some issues in the reference implementation. I propose to do it separately.

Comment (ilya-lavrenov) [reply inference-engine/src/legacy_api/src/graph_transformer.cpp:224]: Ok, please, create a ticket for this.

**Review by lazarevevgeny — APPROVED** [review]
The PR solves the issue with inference with my DETR model.

**Pull request merged**